mkdocs/docs/HPC/connecting.md

```diff
@@ -6,8 +6,9 @@ Before you can really start using the {{ hpc }} clusters, there are several thin
 you need to do or know:
 
 1.  You need to **log on to the cluster** using an SSH client to one of
-    the login nodes. This will give you command-line access. The
-    software you'll need to use on your client system depends on its
+    the login nodes or by using the [HPC web portal](web_portal.md). 
+    This will give you command-line access. 
+    The software you'll need to use on your client system depends on its
     operating system.
```

Comment:
Also mention here that a standard web browser like Firefox or Chrome is sufficient when using the web portal?